File: fuelSpent.cpp
#include <stdio.h>
#include <iostream>
#include <iomanip>

int main() {
    int timing, speedAvarage;
    double fuelNedded;

    std::cin >> timing;
    std::cin >> speedAvarage;
    fuelNedded = static_cast<double>(timing * speedAvarage) / 12;
    std::cout << std::fixed << std::setprecision(3) << fuelNedded << std::endl;


    return 0;

};

File: fuelSpent.go
package main
import (
	"fmt"
)

func main() {
	var timing, speedAvarage int;
	var fuelNedded float64;


	fmt.Scanln(&timing);
	fmt.Scanln(&speedAvarage);
	fuelNedded = float64(timing * speedAvarage) / 12.0;

	fmt.Printf("%.3f\n", fuelNedded);
}